data_diff/diff_tables.py

```diff
@@ -85,6 +85,7 @@ class DiffStats:
     table2_count: int
     unchanged: int
     diff_percent: float
+    extra_column_diffs: Dict[str, int]
```

Comment:
Since we're adding a parameter here, we'll need to account for it in the other DiffStats() in this file

Ex: adding `, None` on line 125

Otherwise they'll throw an exception